x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/transform/DataFrameTransformConfig.java

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-package org.elasticsearch.xpack.dataframe.transforms;
+package org.elasticsearch.xpack.core.dataframe.transform;
```

Comment:
It looks like there's a discrepancy here now where the package is plural in the `dataframe` plugin and singular in the `core` plugin.  I think it would be good to be consistent.

Reply:
Good spot, the core classes are in the `transform` package whereas the dataframe package is `transforms`. Should the name be plural because this is a collection of classes? @hendrikmuhs what do you think?

Reply:
I see, my bad. I would go for `transforms`. As you said: there will be more, e.g. `org.elasticsearch.xpack.core.dataframe.transforms.transaction`

Reply:
👍 transforms it is